test/drenv/drenv_test.py

```diff
@@ -18,6 +18,21 @@ def test_start(tmpenv):
     assert cluster.status(tmpenv.prefix + "cluster") == cluster.READY
 
 
+def test_dump_unknown():
+    """
+    succeeds as dump shows how drenv binds templates, expands scripts, arguments,
+    name workers and applies default values with respect to .yaml argument
+    even if the cluster is not present
+    """
+    commands.run("drenv", "dump", "--name-prefix", "unknown-", "external.yaml")
+
+
+def test_dump(tmpenv):
+    # succeeds when the cluster is ready as well, dumps wrt .yaml passed
+    commands.run("drenv", "dump", "--name-prefix", tmpenv.prefix, "external.yaml")
+    assert cluster.status(tmpenv.prefix + "cluster") == cluster.READY
```

Comment:
dump does not access the cluster so we don't need to depend on tmpenv, or test that 
the cluster is ready after dumping. So we want to test 2 cases:

- dump without prefix
- dump with prefix

What we really want to test, is that dump actually dump the right json, so we want
to get the output of the drenv command, parse it as json, and validate the json.

Also we don't need to test the external yaml for dump, we can use any yaml, maybe
the example yaml? 

Example:

     out = commands.run("drenv", "dump", "example.yaml")
     dump = json.loads(out)
     assert dump["profiles"][0]["name"] == "ex1"
     ...

Same test with prefix "test-" will show:

    assert dump["profiles"][0]["name"] == "test-ex1"